### ISSUE TITLE
.NET crash on upscaling video after frame processing

### ISSUE THREAD

**Issue by tallpgr** [n00mkrad/cupscale#36]
Upscaling a short WEBM (vp9/opus) video with 4xFSDedither_Riven.pth. Original file attributes:
`Input #0, matroska,webm, from 'xxx.webm':
  Metadata:
    ENCODER         : Lavf58.76.100
  Duration: 00:01:26.08, start: -0.003000, bitrate: 332 kb/s
    Stream #0:0: Video: vp9 (Profile 0), yuv420p(tv, progressive), 640x427, SAR 1:1 DAR 640:427, 15 fps, 15 tbr, 1k tbn, 1k tbc (default)
    Metadata:
      ENCODER         : Lavc58.134.100 libvpx-vp9
      DURATION        : 00:01:26.070000000
    Stream #0:1: Audio: opus, 48000 Hz, stereo, fltp (default)
    Metadata:
      ENCODER         : Lavc58.134.100 opus
      DURATION        : 00:01:26.084000000
`
At the conclusion of upscaling the frames into img-out, Cupscale reports (via a popup window) this exception:

**Comment by n00mkrad** [comment]
Is your output mode set to "same as source"?

If it is, try changing it to MP4.

**Comment by n00mkrad** [comment]
Yeah but that currently only works for GIF and MP4. I thought WEBM would fall back to MP4 instead of showing an error though.
